frontend/javascripts/oxalis/geometries/materials/plane_material_factory.ts

```diff
@@ -148,6 +155,9 @@ class PlaneMaterialFactory {
       globalMousePosition: {
         value: new THREE.Vector3(0, 0, 0),
       },
+      activeSegmentPosition: {
+        value: new THREE.Vector3(0, 0, 0),
```

Comment:
will this show a crosshair at 0, 0, 0 when switching to the proofreading tool initially?

Reply:
Yep it is if the bounding box starts at 0, 0, 0. There's a small party going on there, another guest is the center dot of the quick select tool (took me some time to figure out where the dot comes from) :grin: 

I'll take care of both issues :)